index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1:

Reply:
eslint now enabled.all errors fixed and disabled files removed.